apps/judicial-system/api/src/app/data-sources/backend.ts

```diff
@@ -93,6 +95,14 @@ export class BackendApi extends DataSource<{ req: Request }> {
     })
   }
 
+  private patch<TBody, TResult>(route: string, body: TBody): Promise<TResult> {
```

Comment:
Nice. in fact, all of our updates should use PATCH rather than PUT, but that is a different issue.